File: C++/1009.cpp
#include <iostream>
using namespace std;

int main()
{
    char name[25];
    double salary;
    double sales;
    cin >> name >> salary >> sales;
    if (sales > 0)
    {
        cout.precision(2);
        cout << fixed << "TOTAL = R$ " << salary + (sales * 0.15) << endl;
    }
    else
    {
        cout.precision(2);
        cout << fixed << "TOTAL = R$ " << salary << endl;
    }

    return 0;
}

File: JavaScript/1009.js
var input = require("fs").readFileSync("/dev/stdin", "utf8");
var lines = input.split("\n");

var person_name = lines.shift();
var salary = parseFloat(lines.shift());
var sales = parseFloat(lines.shift());

if (sales > 0) {
  console.log("TOTAL = R$", (salary + sales * 0.15).toFixed(2));
} else {
  console.log("TOTAL = R$", salary.toFixed(2));
}